tests/bluetooth/audio/ascs/src/test_common.c

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/fff.h>
+#include <zephyr/types.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/sys/util_macro.h>
+
+#include "bap_unicast_server.h"
+#include "bap_stream.h"
+#include "gatt_expects.h"
+
+#include "test_common.h"
+
+static uint8_t attr_found(const struct bt_gatt_attr *attr, uint16_t handle, void *user_data)
+{
+	const struct bt_gatt_attr **result = user_data;
+
+	*result = attr;
+
+	return BT_GATT_ITER_STOP;
+}
+
+const struct bt_gatt_attr *test_ase_control_point_get(void)
+{
+	static const struct bt_gatt_attr *attr;
+
+	if (attr == NULL) {
+		bt_gatt_foreach_attr_type(0x0001, 0xFFFF, BT_UUID_ASCS_ASE_CP, NULL, 1,
+					  attr_found, &attr);
+	}
+
+	zassert_not_null(attr, "ASE Control Point not found");
+
+	return attr;
+
+}
+
+uint8_t test_ase_get(const struct bt_uuid *uuid, int num_ase, ...)
```

Comment:
Can you elaborate on why we are using va here, instead of `struct bt_gatt_attr **attr`?

Reply:
The function allows to get multiple ASE attributes.
I prepared this function to test configuration of multiple ASE's.

Reply:
Ah, alright :) For the current use it seemed to only be used to get a single attribute, so that's why my suggestion made sense. 

You could still use my suggestion for multiple attrs as well if you had something like
```
uint8_t test_ase_get(const struct bt_uuid *uuid, int num_ase, struct bt_gatt_attr *attr[])
{
	...
}

void foo(const struct bt_uuid *uuid)
{
	struct bt_gatt_attr my_attrs[5];

	test_ase_get(uuid, 5, my_attrs);
}
```